social-network/src/MissedArticle.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import './Article.css';
+import './Author';
+
+
+export default class MissedArticle extends React.Component{
```

Comment:
This is a case where maybe `Article` and `MissedArticle` can be merged into one shared component.